.pylintdict

```diff
@@ -84,6 +84,7 @@ kwarg
 kwargs
 langle
 macos
+main
```

Comment:
`main` is not in the base dictionary - given that's a very common word that's hard to believe

Reply:
> `main` is not in the base dictionary - given that's a very common word that's hard to believe
oh, wow, removed.